dbms/src/Common/getNumberOfPhysicalCPUCores.cpp

```diff
@@ -15,6 +17,22 @@ unsigned getNumberOfPhysicalCPUCores()
 {
 #if defined(__x86_64__)
 
+    std::ifstream cgroup_read_in("/sys/fs/cgroup/cpu/cpu.cfs_quota_us");
```

Comment:
Better to wrap into something like `#if defined(__linux__)` as cgroups exist only on Linux.

Reply:
Done